android_sample/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
This should probably be 3.1 to match the min version in the top level cmake file.